packages/trader/src/Modules/Trading/Containers/trade.tsx

```diff
@@ -0,0 +1,235 @@
+            setTimeout(() => setTrySyntheticIndices(false));
+        }
+    };
+
+    const topWidgets = React.useCallback(
+        () => <ChartTopWidgets open_market={open_market} open={try_synthetic_indices || try_open_markets} />,
+        [open_market, try_synthetic_indices, try_open_markets]
+    );
+
+    const form_wrapper_class = is_mobile ? 'mobile-wrapper' : 'sidebar__container desktop-only';
+    const chart_height_offset = React.useMemo(() => {
+        if (is_accumulator) return '295px';
+        if (is_turbos) return '300px';
+        return '259px';
+    }, [is_turbos, is_accumulator]);
+
+    return (
+        <div
+            className={classNames('trade-container', {
+                [`trade-container--${is_accumulator ? 'accumulators' : 'turbos'}`]: is_accumulator || is_turbos,
+            })}
+            id='trade_container'
+        >
+            <DesktopWrapper>
+                <PositionsDrawer />
+            </DesktopWrapper>
+            {/* Div100vhContainer is workaround for browsers on devices
+                    with toolbars covering screen height,
+                    using css vh is not returning correct screen height */}
+            <Div100vhContainer
+                className='chart-container'
+                height_offset={chart_height_offset}
+                id='chart_container'
+                is_disabled={isDesktop()}
+            >
+                <NotificationMessages />
+                <React.Suspense
+                    fallback={<ChartLoader is_dark={is_dark_theme} is_visible={!symbol || !!is_chart_loading} />}
+                >
+                    <DesktopWrapper>
+                        <div className={classNames('chart-container__wrapper', { 'vanilla-trade-chart': is_vanilla })}>
+                            <ChartLoader is_visible={is_chart_loading || should_show_active_symbols_loading} />
+                            <TradeChart topWidgets={topWidgets} is_accumulator={is_accumulator} />
+                        </div>
+                    </DesktopWrapper>
+                    <MobileWrapper>
+                        <ChartLoader is_visible={is_chart_loading || should_show_active_symbols_loading} />
+                        <SwipeableWrapper
+                            className={classNames({ 'vanilla-trade-chart': is_vanilla })}
+                            is_disabled={
+                                !show_digits_stats ||
+                                !is_trade_enabled ||
+                                form_components.length === 0 ||
+                                is_chart_loading ||
+                                should_show_active_symbols_loading
+                            }
+                            is_swipe_disabled={swipe_index === 1}
+                            onChange={onChangeSwipeableIndex}
+                        >
+                            {show_digits_stats && <DigitsWidget digits={digits} tick={tick} />}
```

Comment:
`digits` and `tick` can be passed like that, same on line 206, 208, 217, 218 and etc.
Ignore if not relevant

Reply:
@kate-deriv unfortunately, `digits` and `tick` will mean `digits={true}` and `tick={true}`. Hence, inapplicable

Reply:
Sorry, yep, it's not an object, my mind went blank) Will resolve other comments)